templates/ts-empty/.actor/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Specify the base Docker image. You can read more about
```

Comment:
I'd say this should be using multi-stage build as the other TS templates, otherwise you would have to install TS as a regular dependency (or install all deps, the way it is now wil omit dev deps)